a3p-integration/proposals/z:acceptance/governance.test.js

```diff
@@ -56,6 +56,17 @@ test.serial(
     }
 
     await governanceDriver.waitForElection();
+
+    const vaultFactoryParamsAfter = await readLatestHead(
+      'published.vaultFactory.governance',
+    );
+
+    t.is(
+      // @ts-expect-error it's a record
+      vaultFactoryParamsAfter.current.ChargingPeriod.value,
+      400n,
```

Comment:
Are we sure it wasn't 400n before?

Reply:
also: we talked about revising `governanceDriver.waitForElection()` to pay attention to which question is relevant.